<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resultText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>900</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 12pt &quot;Poppins SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Cipherteks :</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="results">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>900</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 8pt &quot;Poppins Medium&quot;;</string>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Enkripsi Pesan</string>
    </property>
   </widget>

   <widget class="QTextBrowser" name="key_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>900</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>

   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>56</y>
      <width>900</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 20pt &quot;Poppins SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>One-Time Pad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>56</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Balik ke awal</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>900</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 12pt &quot;Poppins SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Plainteks</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>150</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>520</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 20pt &quot;Poppins SemiBold&quot;;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="text">
      <string>Pilih Teknik</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>104</y>
       <width>520</width>
       <height>200</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="encryptButton">
        <property name="text">
         <string>Enkripsi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decryptButton">
        <property name="text">
         <string>Dekripsi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>900</width>
      <height>120</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="key_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>900</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 12pt &quot;Poppins SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Kunci</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
